src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -384,6 +385,14 @@ const formConfig = {
           uiSchema: hospitalizationHistory.uiSchema,
           schema: hospitalizationHistory.schema,
         },
+        unemployabilityCertification: {
+          title: 'Certification',
```

Comment:
This title shows up on the review page, maybe use something more related to unemployability?